napari/utils/events/containers/_nested_list.py

```diff
@@ -184,7 +185,9 @@ def __getitem__(self, key: MaybeNestedIndex):
         if isinstance(key, tuple):
             item: NestableEventedList[_T] = self
             for idx in key:
-                item = item[idx]  # type: ignore
+                if not isinstance(item, MutableSequence):
+                    raise IndexError(f'index out of range: {key}')
+                item = item[idx]
             return item
```

Comment:
Shouldn't this be just any sequence (including non-mutables)?

Reply:
I guess it depends if we want to allow things like:

```python
nested_list = [1, [2, (3, 4, 5)]]
nested_list [1, 1, 2] == 4
```

That tuple is unlikely to be a true member of the Tree ... Someone could still do `nested_list[1, 1][2]` of course.  thoughts?

Reply:
I guess you're right, I feel like most of the times if this happened it would be a mistake.